### PR TITLE
issues: add tag-input suggester background fixes #670

### LINKED ISSUE #670: repo tag-input suggester has no background
* **Browser**: FF 61
* **Operating System**: win 10 pro
* **Screenshot**:

tag-input

Ill have a proposed PR fix shortly

* **HTML of the section where the issue occurs**:

````html
<div class="tag-input form-control d-inline-block bg-white py-0">
          <ul class="js-tag-input-selected-tags d-inline">
            <li class="d-none topic-tag-action f6 float-left js-tag-input-tag js-template">
              <span class="js-placeholder-tag-name"></span>
              <button type="button" class="delete-topic-button f5 no-underline ml-2 js-remove" tabindex="-1">
                ×
              </button>
              <input name="repo_topics[]" class="js-topic-input" value="" type="hidden">
            </li>

          <li class="topic-tag-action f6 float-left js-tag-input-tag">
              appveyor
              <button type="button" class="delete-topic-button f5 no-underline ml-2 js-remove" tabindex="-1">
                ×
              </button>
              <input name="repo_topics[]" class="js-topic-input" value="appveyor" type="hidden">
            </li><li class="topic-tag-action f6 float-left js-tag-input-tag">
              appveyor-dark
              <button type="button" class="delete-topic-button f5 no-underline ml-2 js-remove" tabindex="-1">
                ×
              </button>
              <input name="repo_topics[]" class="js-topic-input" value="appveyor-dark" type="hidden">
            </li></ul>

          <auto-complete src="/style-it-themes/appveyor-ci-inspired-dark/topic_autocomplete" aria-owns="repo-topic-popup" role="combobox" aria-haspopup="listbox" aria-expanded="true" open="">
            <input id="repo_topics" class="tag-input-inner form-control bg-white shorter d-inline-block p-0 my-1 border-0" autocomplete="off" autofocus="" spellcheck="false" aria-autocomplete="list" aria-controls="repo-topic-popup" type="text">
            <div class="js-navigation-container js-active-navigation-container">
              <ul class="suggester border width-full bg-white left-0" id="repo-topic-popup" role="listbox" style="position: absolute; width: 130px; top: 37px;">
  <li role="option" class="js-navigation-item border-bottom text-bold py-1 px-3" data-autocomplete-value="api" id="repo-topic-popup-option-0" aria-selected="false">api</li>
  <li role="option" class="js-navigation-item border-bottom text-bold py-1 px-3" data-autocomplete-value="app" id="repo-topic-popup-option-1" aria-selected="false">app</li>
  <li role="option" class="js-navigation-item border-bottom text-bold py-1 px-3" data-autocomplete-value="api-client" id="repo-topic-popup-option-2" aria-selected="false">api-client</li>
  <li role="option" class="js-navigation-item border-bottom text-bold py-1 px-3" data-autocomplete-value="application" id="repo-topic-popup-option-3" aria-selected="false">application</li>
  <li role="option" class="js-navigation-item border-bottom text-bold py-1 px-3" data-autocomplete-value="apache" id="repo-topic-popup-option-4" aria-selected="false">apache</li>
  <li role="option" class="js-navigation-item border-bottom text-bold py-1 px-3" data-autocomplete-value="api-wrapper" id="repo-topic-popup-option-5" aria-selected="false">api-wrapper</li>
  <li role="option" class="js-navigation-item border-bottom text-bold py-1 px-3" data-autocomplete-value="api-rest" id="repo-topic-popup-option-6" aria-selected="false">api-rest</li>
  <li role="option" class="js-navigation-item border-bottom text-bold py-1 px-3" data-autocomplete-value="apple" id="repo-topic-popup-option-7" aria-selected="false">apple</li>
  <li role="option" class="js-navigation-item border-bottom text-bold py-1 px-3" data-autocomplete-value="api-server" id="repo-topic-popup-option-8" aria-selected="false">api-server</li>
  <li role="option" class="js-navigation-item border-bottom text-bold py-1 px-3 navigation-focus" data-autocomplete-value="apollo" id="repo-topic-popup-option-9" aria-selected="true">apollo</li>
</ul>
            </div>
          </auto-complete>
        </div>
````

### PR DESCRIPTION
**Pull request by the-j0k3r**
after
tag-input-fix

### CODE REVIEW

File: github-dark.css [hunk 1]
```diff
@@ -3828,7 +3828,7 @@
   .zhc-pr-issue-connector__info, .zhu-tag-default, .zhc-chart-container__header,
   .zhc-issues-list__header, .zhc-toggleable--is-disabled,
   .zhc-item-table__row:hover, .zh-collapse-control__main-icon,
-  .zhc-sidebar-profile {
+  .zhc-sidebar-profile, .tag-input .suggester {
```
Comment (Mottie) [inline github-dark.css]: Please move this out of the ZenHub (`.zhc`) section and up to around line 1913 where the background color is `#181818` and border is `#343434` or `#484848` whichever looks better.

Comment (the-j0k3r) [reply github-dark.css]: @Mottie done and done

**Comment by Mottie** [comment]
Thanks!

**Comment by the-j0k3r** [comment]
Bitte

**Pull request merged**